development/code/Game.ts

```diff
@@ -59,11 +60,16 @@ let deathCounter = 0;
 let isCastling = false;
 let isFriendlyFire = false;
 let isPieceKilled = false;
+let itemPlacedDownRecently = false;
 let fellOffTheBoardPiece: Piece | undefined;
 let isGameFinished = false;
 
 function initializeGame() {
   rulesManager = new RulesManager();
+
+  players.forEach((player) => {
+    initialiseInventoryUI(player.color);
```

Comment:
On land and sky, the British shall cry!!

Reply:
Ok and?